src/boot/efi/proto/tcg.h

```diff
@@ -11,6 +11,7 @@
 #define TCG_ALG_SHA 0x4
 #define EFI_TCG2_EVENT_HEADER_VERSION 1
 #define EV_IPL 13
+#define EV_EVENT_TAG 0x6
```

Comment:
it's a bit weird that even through EV_IPL and EV_EVENT_TAG are values for the same field one is encoded in hex, and one in dec. please convert both to hex or both to dec...

also, maybe change this to be unsigned values, given the fields for this are unsigned too

Reply:
UINT32_C() maybe even